boilerplate/update

```diff
@@ -37,7 +37,7 @@ else
   echo "Updating the update script."
   rsync -a "${OUT}/boilerplate/update" "$0"
   echo "Copying utilities"
-  rsync -a -r ${OUT}/boilerplate/_* $DIR
+  rsync -a -r --delete ${OUT}/boilerplate/_* $DIR
```

Comment:
Probably should be a separate PR.

Reply:
I've made #46 and fixup this PR once that is merged.

Reply:
Thanks!